students/skowronski_rafal/lesson_02_flow_control/lesson_03/problem_11.py

```diff
@@ -0,0 +1,16 @@
+# Solves problem 11 - chocolate bar
+
+
+def _main():
+    n = int(input('Enter length of the first side of the chocolate (n)): '))
+    m = int(input('Enter length of the second side of the chocolate (m)): '))
+    k = int(input('Enter number of squares (k): '))
+
+    is_split_possible = (k / n < m and k % n == 0) or \
```

Comment:
This is unreadable, You should change 'n', 'm', 'k' variable names. It will be more clear